### PR TITLE
Add option to add sidekiq

### LINKED ISSUE #3: Option: add Sidekiq
- gems
- `Jenkinsfile`
- `Dockerfile`
- config file
- `Procfile`
- test helpers
- ...

### PR DESCRIPTION
**Pull request by martijnversluis**
# Functional

- run `launch_base new --with-sidekiq` to add Sidekiq to a new Rails app
- run `launch_base add sidekiq` to add Sidekiq to an existing Rails app

# Technical

You can now easily add plugins. Just create a file in `lib/launch_base/plugins` containing a class that inherits from `LaunchBase::Plugin`. When running `launch_base add <YOUR_PLUGIN>` or `launch_base new --with-<YOUR_PLUGIN>` the `install` method of your plugin will be executed. 

Fixes #3

### CODE REVIEW

File: lib/launch_base/cli/base.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+      private
+
+      def show_banner
+        say "\u{1f680} Kabisa #{LaunchBase}\n\n"
+      end
+
+      def install_modules(project_path, options)
+        Plugin.each_plugin do |plugin_name, plugin|
+          if options["with-#{plugin_name.tr('_', '-')}"]
```
Comment (nickve28) [inline lib/launch_base/cli/base.rb]: can we add this `"with-#{plugin_name.tr('_', '-')}"` stuff in a separate method?, eg `def sanitized_plugin_name`

File: lib/launch_base/plugin.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      @plugins[name] = plugin
+    end
+
+    def self.each_plugin(&block)
+      @plugins.each(&block)
+    end
+
+    def self.plugin_name
+      class_name
+        .gsub(/([A-Z\d]+)([A-Z][a-z])/, '\1_\2')
```
Comment (nickve28) [inline lib/launch_base/plugin.rb]: a regex has no name

File: lib/launch_base/cli/base.rb [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      end
+
+      desc 'new', "Create a new #{LaunchBase} project"
+      long_desc <<-LONGDESC
+        `#{@package_name} new [project-path]` creates a new rails project with Kabisa preferences.
+      LONGDESC
+      Plugin.each_plugin do |plugin_name, _plugin|
+        option "with-#{plugin_name.tr('_', '-')}", type: :boolean, default: false
+      end
+      def new(project_path)
```
Comment (drumusician) [inline lib/launch_base/cli/base.rb]: maybe give this some air to breath. It feels a little cluttered

**Review by drumusician — APPROVED** [review]
I guess after some small code style improvements go ahead and merge this ;)

**Pull request merged**